commands/eval.md

```diff
@@ -12,6 +12,13 @@ to ensure the correct execution of scripts, both in standalone and clustered dep
 The script **should only** access keys whose names are given as input arguments.
 Scripts **should never** access keys with programmatically-generated names or based on the contents of data structures stored in the database.
 
+**Important:**
+in some cases, users will abuse lua eval.
+Each `EVAL` call generates a new lua script, which is added to the lua interpreter and cached to redis-server, consuming a large amount of memory over time.
+Since `EVAL` is mostly the one that abuses the lua cache, and these won't have pipeline issues (i.e. the script won't disappear unexpectedly, and cause errors like it would with `SCRIPT LOAD` and `EVALSHA`), we implement a plain FIFO LRU eviction only for these (not for scripts loaded with `SCRIPT LOAD`).
+Starting from Redis 8.0, `EVAL` SCRIPTS will maintain an LRU list of length 500, when the number exceeds the limit, the oldest `EVAL` script will be evicted.
+The number of evicted scripts can be viewed through `INFO`'s `evicted_scripts`.
```

Comment:
```suggestion
in some cases, users will abuse Lua EVAL by embedding values in the script instead of providing them as argument, and thus generating a different script on each call to EVAL.
These are added to the Lua interpreter and cached to redis-server, consuming a large amount of memory over time.
Starting from Redis 8.0, scripts loaded with `EVAL` will be deleted from redis after a certain number (least recently used order).
The number of evicted scripts can be viewed through `INFO`'s `evicted_scripts`.
```